pisa/core/distribution_maker.py

```diff
@@ -87,17 +99,75 @@ def __init__(self, pipelines, label=None):
             if not isinstance(pipeline, Pipeline):
                 pipeline = Pipeline(pipeline)
             self._pipelines.append(pipeline)
+
+        data_run_livetime = None
+        if set_livetime_from_data:
+            #
+            # Get livetime metadata if defined in any stage in any pipeline
+            #
+            for pipeline_idx, pipeline in enumerate(self):
+                for stage_idx, stage in enumerate(pipeline):
+                    if not (
+                        hasattr(stage, "metadata")
+                        and isinstance(stage.metadata, Mapping)
+                        and "livetime" in stage.metadata
+                    ):
+                        continue
+
+                    if data_run_livetime is None:
+                        data_run_livetime = stage.metadata["livetime"]
+
+                    if stage.metadata["livetime"] != data_run_livetime:
+                        raise ValueError(
+                            "Pipeline index {}, stage index {} has data"
+                            " livetime = {}, in disagreement with"
+                            " previously-found livetime = {}".format(
+                                pipeline_idx,
+                                stage_idx,
+                                stage.metadata["livetime"],
+                                data_run_livetime,
+                            )
+                        )
+            #
+            # Set param `params.livetime` for any pipelines that have it
+            #
+            if data_run_livetime is not None:
+               # data_run_livetime *= ureg.sec
```

Comment:
Why did you remove units here? Have you figured out how to handle units elsewhere? (This would be preferred to always using seconds, so removing this should eventually be the goal, but I expect it will take a little more work to make it work...)

Reply:
As noted above, it caused issues when comparing the livetime obtained from the files with the livetime obtained from the config file (since the latter are defined so far as dimensionless). We could very well put it back in, if it's possible to define units at the config file level

Reply:
ok, the units are now back